starter-code/css/main.css

```diff
@@ -6,3 +6,132 @@ Green: #00B172
 White: #FFF
 
 */
+.body{
```

Comment:
to add font (and make your website look better ;) ) you can use this piece of code, basically it's calling a google font and it allow you to use it anywhere you want on your website : 

```css

@import url("https://fonts.googleapis.com/css?family=Raleway");
@import url("https://fonts.googleapis.com/css?family=Montserrat:400,700");

* {
  margin: 0;
  padding: 0;
  font-family: MOntserrat;
}

```